### PR TITLE
Arbitrator republish signedWitnesses on startup

### PR DESCRIPTION
**Pull request by sqrrm**
This republishes all signed witnesses 60 seconds after boostrap complete for arbitrators

### CODE REVIEW

File: core/src/main/java/bisq/core/app/BisqSetup.java [hunk 1]
```diff
@@ -692,6 +692,9 @@ private void initDomainServices() {
 
         accountAgeWitnessService.onAllServicesInitialized();
         signedWitnessService.onAllServicesInitialized();
+        if (user.getRegisteredArbitrator() != null) {
```
Comment (chimp1984) [inline core/src/main/java/bisq/core/app/BisqSetup.java]: Why not do it inside of signedWitnessService? User can be injected. Better to not spread out responsibilities...

**Review by chimp1984 — CHANGES_REQUESTED** [review]

File: core/src/main/java/bisq/core/app/BisqSetup.java (hunk 1 above)
Comment (sqrrm) [reply core/src/main/java/bisq/core/app/BisqSetup.java]: Yes, perhaps that's neater. Fixed

**Review by chimp1984 — COMMENTED** [review]
utACK

**Review by ripcurlx — APPROVED** [review]
utACK

**Pull request merged**